src/test/java/HW20.java

```diff
@@ -16,7 +16,7 @@ public class HW20 extends BaseTest{
     public void DeletePlayList(){
```

Comment:
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.

For example `public void deletePlayList()`

and 

**According to the Java style guide:** 

1. Class names should be nouns, in mixed cases with the first letter of each internal word capitalized. Interfaces names should also be capitalized just like class names.
2. Use whole words and must avoid acronyms and abbreviations.

For example `public class HomeWork20`